static/app/plugins/components/pluginIcon.tsx

```diff
@@ -138,9 +145,28 @@ const PluginIcon = styled('div')<Props>`
     (pluginId !== undefined && ICON_PATHS[pluginId]) || DEFAULT_ICON});
 `;
 
+const PluginIcon = ({pluginId, size, sentryApp, isColor, className}: Props) => (
```

Comment:
@leeandher The plugin icon should be agnostic to the type of integration you are using; it should have no concept of a `sentryApp`.